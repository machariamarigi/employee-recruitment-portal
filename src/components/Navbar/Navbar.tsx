import React from 'react'

export default function Navbar() {
    return (
        <nav className="flex flex-wrap items-center justify-between p-5 bg-blue-800">
           <p className="text-2xl font-bold text-white">Employee Recruitment Portal</p>
        </nav>
    )
}
