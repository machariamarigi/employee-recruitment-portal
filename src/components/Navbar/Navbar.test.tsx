import React from 'react'
import { shallow } from 'enzyme';
import Navbar from './Navbar'

describe('Navbar Testing', () => {
    it('should render the navbar with the title of the app', () => {
        const wrapper = shallow(<Navbar />)
        expect(wrapper).toMatchSnapshot()
    })
})
