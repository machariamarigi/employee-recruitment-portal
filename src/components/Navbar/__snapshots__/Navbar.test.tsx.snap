// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Testing should render the navbar with the title of the app 1`] = `ShallowWrapper {}`;
